src/app/public/modules/action-hub/action-hub.component.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Component,
+  Input,
+  OnDestroy
+} from '@angular/core';
+
+import {
+  Observable,
+  Subject
+} from 'rxjs';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  SkyLink
+} from '../link-list/types/link';
+
+import {
+  NeedsAttention
+} from '../needs-attention/types/needs-attention';
+
+import {
+  Configuration
+} from './types/configuration';
+
+@Component({
+  selector: 'sky-action-hub-buttons',
+  template: `<ng-content></ng-content>`
+})
+export class SkyActionHubButtonsComponent {}
+
+@Component({
+  selector: 'sky-action-hub-content',
+  template: `<ng-content></ng-content>`
+})
+export class SkyActionHubContentComponent {}
+
+@Component({
+  selector: 'sky-action-hub',
+  templateUrl: './action-hub.component.html'
+})
+export class SkyActionHubComponent implements OnDestroy {
+  @Input()
+  public set config(value: Configuration | Observable<Configuration>) {
+    if (value instanceof Observable) {
+      value
+        .pipe(takeUntil(this.ngUnsubscribe))
+        .subscribe(configuration => {
+          this._configuration = configuration;
+        });
+    } else {
+      this._configuration = value;
+    }
+  }
+
+  @Input()
+  public configAuth = false;
+
+  public loading = true;
```

Comment:
So recognizing that it can be confusing - our style guidelines technically don't call for pure alphabetical ordering of properties. We do 

1. Inputs
2. Inputs with setters/getters
3. Outputs
4. Pure public properties
5. ViewChild/ContentChild properties
6. Private properties
7. Private properties accessed via getters/setters

https://app.blackbaud.com/skyux-team-docs/style#class-properties